client/src/components/History/Content/model/states.js

```diff
@@ -74,3 +79,8 @@ export const STATES = {
         spin: true,
     },
 };
+
+/** For collections it is necessary to reduce the available states of the collection items to a single state which
+ * can be displayed to represent the collection itself. The order matters.
+ */
+export const STATES_REDUCE = ["error", "failed", "queued", "upload", "running", "paused"];
```

Comment:
```suggestion
export const HIERARCHICAL_COLLECTION_JOB_STATES = ["error", "failed", "queued", "upload", "paused", "running", "new"];
```
paused should have a higher priority than running ... if a single dataset is paused, you know that you need to do something in order to be able to use that collection. And there should also be a `new` state, I think.